srml/support/src/dispatch.rs

```diff
@@ -37,7 +37,7 @@ pub type Result = result::Result<(), &'static str>;
 /// A lazy call (module function and argument values) that can be executed via its dispatch()
 /// method.
 pub trait Dispatchable {
-	/// Every function call to your runtime has an origin which specifies where the extrinsic was
+	/// Every function call to your runtime has an origin, which specifies where the extrinsic was
```

Comment:
Would it be "_from_ your runtime"? A call goes from the runtime to the SRML, no?

Reply:
oops thanks!